src/dbc.rs

```diff
@@ -10,10 +10,15 @@ use crate::{DbcContent, Denomination, Error, KeyManager, PublicKey, Result, Sign
 use serde::{Deserialize, Serialize};
 use tiny_keccak::{Hasher, Sha3};
 
+/// mint_root_public_key is the root pubkey, from which
+/// denomination-specific pubkeys are derived.
+///
+/// mint_signature is a signature that can be verified with:
+/// mint_root_public_key.derive_child(&self.denomination().to_bytes()).verify(self.mint_signature)
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Deserialize, Serialize)]
 pub struct Dbc {
     pub content: DbcContent,
-    pub mint_public_key: PublicKey,
+    pub mint_root_public_key: PublicKey,
```

Comment:
Again, `mint_public_key` was fine, arguably we were putting a `mint_denomination_key` here instead of the `mint_public_key`

Reply:
👍 on `mint_denomination_signature`